src/main/java/seedu/address/logic/commands/SortMCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class SortMCommand extends Command {
```

Comment:
I think it would be better to refactor this to `SortMeetingCommand` since standardising the command names would make it easier for others to understand the code.